.travis.yml

```diff
@@ -1,27 +1,26 @@
 sudo: required
 os:
-  - linux
-  - osx
+- linux
+- osx
 language: node_js
 node_js:
-  - node
-  - '9'
-  - '8'
-  - '7'
-  - '6'
+- node
+- '9'
+- '8'
+- '7'
+- '6'
 cache:
   directories:
   - "$HOME/.npm"
 before_install:
-  - npm install -g npm@latest
+- openssl aes-256-cbc -K $encrypted_0f9bbf7a60f4_key -iv $encrypted_0f9bbf7a60f4_iv
```

Comment:
Be sure to describe exactly what this command does in the README.
It looks very complicated.